bugsy/bugsy.py

```diff
@@ -90,6 +90,21 @@ def __init__(
                 else:
                     raise LoginException(result['message'])
 
+    def bug_url(self, bug_number):
```

Comment:
Why have this here instead of on the `bug`? Going on your initial comment it would seem the flow would be better suited to
```python
bug.add_comment('zomg I pushed... my rev is ')
print bug.url
```